docs/DeveloperGuide.md

```diff
@@ -29,22 +29,29 @@ This product will make recruiters’ lives easier through categorisation and fil
   * [Common classes](#common-classes)
 - [**Implementation**](#implementation)
   * [Add feature](#add-feature)
-  * [Datetime for interview](#datetime-for-interview)
   * [Edit feature](#edit-feature)
-  * [Filter interview feature](#filter-interview-feature)
     + [Design considerations:](#design-considerations)
   * [Find feature](#find-feature)
+    + [Design considerations:](#design-considerations-1)
+  * [Filter interview feature](#filter-interview-feature)
+    + [Design considerations:](#design-considerations-2)
+  * [Delete feature](#delete-feature)
```

Comment:
In User Guide, the order is ... `edit` -> `delete` -> `find` -> `filter_interview` -> `show` ... Should we move `delete` to be after `edit` so that two guides are consistent?

Reply:
The User Guide table of contents seems to be edit -> find -> filter_interview -> delete